lib/send.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+// utility to send messages to processes
+function send(ps, name, data) {
+	if (typeof ps === 'string') {
```

Comment:
Turns out it will conflict with the following:

```js
send('teardown', ps);
```

where events don't have any data.